python/lsst/meas/algorithms/sourceSelector.py

```diff
@@ -299,26 +355,25 @@ def apply(self, catalog):
         """
         selected = np.ones(len(catalog), dtype=bool)
         value = catalog[self.name]
-        if self.minimum is not None:
-            selected &= value > self.minimum
-        if self.maximum is not None:
-            selected &= value < self.maximum
-        return selected
+        return BaseLimit.apply(self, value)
 
 
 class ScienceSourceSelectorConfig(pexConfig.Config):
     """Configuration for selecting science sources"""
     doFluxLimit = pexConfig.Field(dtype=bool, default=False, doc="Apply flux limit?")
     doFlags = pexConfig.Field(dtype=bool, default=False, doc="Apply flag limitation?")
     doUnresolved = pexConfig.Field(dtype=bool, default=False, doc="Apply unresolved limitation?")
+    doSignalToNoise = pexConfig.Field(dtype=bool, default=False, doc="Apply signal-to-noise limit?")
     fluxLimit = pexConfig.ConfigField(dtype=FluxLimit, doc="Flux limit to apply")
     flags = pexConfig.ConfigField(dtype=RequireFlags, doc="Flags to require")
     unresolved = pexConfig.ConfigField(dtype=RequireUnresolved, doc="Star/galaxy separation to apply")
+    signalToNoise = pexConfig.ConfigField(dtype=SignalToNoiseLimit, doc="Signal-to-noise limit to apply")
 
     def setDefaults(self):
         pexConfig.Config.setDefaults(self)
         self.flags.bad = ["base_PixelFlags_flag_edge", "base_PixelFlags_flag_interpolated",
                           "base_PixelFlags_flag_saturated"]
+        self.signalToNoise.fluxField = "base_PsfFlux_flux"
```

Comment:
Do you also need to default the errField (since having them associated is not assumed?)